src/DwitTech.AccountService.Core/Services/UserService.cs

```diff
@@ -44,6 +47,7 @@ IHttpContextAccessor httpContextAccessor
             _configuration = configuration;
             _authenticationService = authenticationService;
             _httpContextAccessor = httpContextAccessor;
+            _mapper = mapper;
```

Comment:
mapper is no longer needed in this class since your implementation has changed. Please remove it along with the using statement, and make the adjustments in the tests to prevent unnecessary addition of a redundant mapper object to the user service constructor in tests going forward.